### PR TITLE
[pytest/test_default_route]: support 4.9 kernel 201911/201811 release

### PR DESCRIPTION
**Pull request by lguohan**
Signed-off-by: Guohan Lu <gulv@microsoft.com>

### Description of PR

Summary:
Fixes # (issue)

### Type of change

- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [x] Test case(new/improvement)

#### How did you do it?
add support for 201911/201811 release

#### How did you verify/test it?
```
johnar@1cafe0c9f994:/data/tests$ py.test $PYTEST_COMMON_OPTS --log-file logs/abc.log route/test_default_route.py  | tee ~/abc.txt
============================= test session starts ==============================
platform linux2 -- Python 2.7.12, pytest-4.6.9, py-1.8.1, pluggy-0.13.1 -- /usr/bin/python
cachedir: .pytest_cache
ansible: 2.8.7
rootdir: /data/tests, inifile: pytest.ini
plugins: ansible-2.2.2
collecting ... collected 2 items

route/test_default_route.py::test_default_route_set_src PASSED           [ 50%]
route/test_default_route.py::test_default_ipv6_route_next_hop_global_address PASSED [100%]
```

### CODE REVIEW

File: tests/route/test_default_route.py [hunk 1]
```diff
@@ -44,18 +57,22 @@ def test_default_route_set_src(duthost):
...
 
     rt = duthost.command("ip -6 route list match ::")['stdout_lines']
     m = re.match(r"^default proto bgp src (\S+)", rt[0])
+    m1 = re.match(r"default via (\S+) dev (\S+) proto 186 src (\S+)", rt[0])
     if m:
         if ipaddress.ip_address(m.group(1)) != lo_ipv6.ip:
             pytest.fail("default route set src to wrong IP {} != {}".format(m.group(1), lo_ipv6.ip))
+    elif m1:
+        if ipaddress.ip_address(m1.group(3)) != lo_ipv6.ip:
+            pytest.fail("default route set src to wrong IP {} != {}".format(m1.group(3), lo_ipv6.ip))
```
Comment (yxieca) [inline tests/route/test_default_route.py:75]: These 2 lines can be merged into a single line:

assertions.pytest_assert(ipaddress.ip_address(m1.group(3)) == lo_ipv6.ip, "default route set src to wrong IP {} != {}".format(m1.group(3), lo_ipv6.ip))

There are more opportunities to condense code above and below in this file.

You do need to import, but this is one line for all the assertions you can use in this file.
from common.helpers import assertions

Comment (lguohan) [reply tests/route/test_default_route.py:75]: I feel current way is more readable

Comment (lguohan) [reply tests/route/test_default_route.py:75]: actually, it cannot because one of m and m1 is none. they cannot be use as the same time.

Comment (yxieca) [reply tests/route/test_default_route.py:75]: you could
assertions.pytest_assert(not m or condition 1, msg1)
asertsions.pytest_assert(not m1 or condition 2, msg2) 

or you can keep you current
if m:
    assertion 1
if m1:
    assertion 2

Comment (lguohan) [reply tests/route/test_default_route.py:75]: prefer to keep it is

Comment (lguohan) [reply tests/route/test_default_route.py]: changed.

**Pull request merged**